requirements.txt

```diff
@@ -1,3 +1,6 @@
 fastapi[standard]==0.115.0
 uvicorn==0.34.0
-pytest==8.3.4
\ No newline at end of file
+pytest==8.3.4
+mypy==1.15.0
+flake8==7.1.1
+multipart==1.2.1
```

Comment:
please move dev dependencies to a separate file, usually it is called requirements-dev.txt
Also multipart is not needed here, it is a transitive requirement
Also add a new line at the end of the file